pkgs/development/ocaml-modules/camomile/1.0.2.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchFromGitHub, buildDunePackage, ocaml, cppo }:
+
+buildDunePackage rec {
+  pname = "camomile";
+  version = "1.0.2";
+
+  duneVersion = "2";
```

Comment:
This is a regression.

Reply:
How so?

Reply:
It used to build fine with the default version of dune.